doc/modules/cassandra/pages/architecture/cql-on-accord.adoc

```diff
@@ -0,0 +1,612 @@
+persistence, and messaging. It also isn’t intended to be a user guide
+and doesn’t fully overlap with the user guide. You should start with the
+user guide to get any context that may be missing here.
+
+== Anatomy of a transaction
+
+The primary way of interacting with Accord is to define a transaction
+using
+https://github.com/apache/cassandra-accord/blob/134df57677bbd5092994923a4dc2f15cd1d033d1/accord-core/src/main/java/accord/primitives/Txn.java#L42[Txn/Txn.InMemory]
+and then asking Accord to execute the transaction. Transactions express
+what they touch by declaring a set of keys or ranges that will be
+read/written to. This set needs to be declared up front and can’t change
+during transaction execution and the transaction can be either a key
+transaction or range transaction but not both.
+
+Range transactions are more expensive for Accord to execute as the
+dependency tracking work Accord has to do is more CPU and memory
+intensive and the transactions are more likely to conflict and block
+execution of other transactions.
+
+Accord is not aware of tables only ranges and keys. Keys and ranges can
+span any tables managed by Accord and the keys and ranges encode the
+tables they apply to. So a range transaction covering multiple tables
+would have a range per table and from Accord’s perspective these are
+completely different ranges.
+
+Transactions also declare a `Kind` which can be `Read`, `Write`
+(Read/Write), `EphemeralRead`, and `ExclusiveSyncPoint`. `Read`, and
+`Write` are what you would expect. `EphemeralRead` is a read that only
+provides per key linearizability, but offers better performance compared
+to `Read` .
+
+`ExclusiveSyncPoint` is transaction that can be used to establish a
+happens before relationship with its dependencies without interfering
+with their execution. `ExclusiveSyncPoint` is used for live migration
+and repair to ensure the visibility at `ALL` of all committed Accord
+transactions to non-transactional reads.
+
+=== Keys and Ranges
+
+`Keys` and `Ranges` are prefixed with `TableId` in the most significant
+position to allow Accord to interact with multiple tables without
+knowing anything about schema. From Accord’s perspective there is just a
+set of ranges that is it responsible for replicating and transacting
+over, and they can be compared, sorted, and split, but beyond that they
+are completely opaque. A follow on effect from this is that token ranges
+(or token ring) are per table.
+
+`Key` is conceptually similar to `DecoratedKey` and is implemented by
+`https://github.com/apache/cassandra/blob/122f5300855d56131948575f80ce0594547c9040/src/java/org/apache/cassandra/service/accord/api/PartitionKey.java[PartitionKey]`
+. `RoutingKey` is conceptually similar to `Token` and is implemented by
+`https://github.com/apache/cassandra/blob/122f5300855d56131948575f80ce0594547c9040/src/java/org/apache/cassandra/service/accord/api/PartitionKey.java[TokenKey]`
```

Comment:
Should be `https://github.com/apache/cassandra/blob/122f5300855d56131948575f80ce0594547c9040/src/java/org/apache/cassandra/service/accord/api/TokenKey.java` (maybe with a different commit)